.circleci/config.yml

```diff
@@ -40,13 +40,15 @@ jobs:
   deploy_development:
     docker:
       - image: circleci/node:10.18.1@sha256:8230f70c03b6ecf521a34125fb090ec4b47068205454e5820859c00260c9bd23
+    environment:
+      ADDON_URL: testing.html
```

Comment:
This doesn't work because the site is built in the `test_and_build` step, using a single config for all. The proper fix for that is will be part of https://github.com/mozilla/FirefoxColor/issues/907

The testing.html page is a useful overview of available xpis for testing, so I don't mind that the Install button is not pointing to the desired page.